src/components/availability-panel/drag-drop-context/DroppableComponent.tsx

```diff
@@ -18,13 +18,22 @@ const DroppableComponent: FC<droppableComponent> = ({
   unAssignedTasks,
   idleMembers,
   isTaskOnDrag,
+  searchTerm,
+  searchTermUnassignedTask,
```

Comment:
Its better to use **searchTermForTask** or **searchTermTask** 
Please change all occurrence of **searchTermUnassignedTask**